### ISSUE TITLE
[Feature Request] Adjust Building Material Strength (Health)

### ISSUE THREAD

**Issue by MiR4i** [valheimPlus/ValheimPlus#137]
In addition to being able to adjust the structural integrity of building material, it would be nice if we could also adjust the strength (health) of it, as well, because sometimes you find that perfect location to build on, but there's a Troll spawn point nearby, and you'd rather not have to re-build a large portion of your wooden building(s) just because a Troll happened to spawn during the few minutes you went AFK and ended up wreaking havoc on something that took you hours to put together.  (Source: Personal Experience)

**Comment by mlschin1** [comment]
I agree with this feature request, I like to build intricate structures and its a real pain to remember how I had each piece if they were to get destroyed. I was hoping for a feature that wouldn't allow buildings to be damaged below 'x'% of their health, so that there are still consequences for not defending your structures, but you don't have to rebuild them. For example, if buildings could be damaged to 1%, but no further, you'd still need to repair them to avoid the damaged look, but not have to completely rebuild.

**Comment by nxPublic** [comment]
This has been implemented with 0.9.0

```ini
[StructuralIntegrity]

; Change false to true to enable this section
enabled=false

; Disables the entire structural integrity system and allows for placement in free air, does not prevent building damage.
disableStructuralIntegrity=false

; Each of these values reduce the loss of structural integrity by % less. The value 100 would result in disabled structural integrity and allow placement in free air.
wood=0
stone=0
iron=0
hardWood=0
```
